spec/commands/inject_spec.rb

```diff
@@ -68,6 +68,13 @@
       str = "gem 'rack-obama', '> 0', :group => [:development]"
       expect(gemfile).to include str
     end
+
+    it "add gem with multiple group in gemfile" do
+      bundle "inject 'rack-obama' '>0' --group=development,test"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'rack-obama', '> 0', :group => [:development, :test]"
```

Comment:
Thanks @segiddins for review. I have edited the line  in this commit https://github.com/bundler/bundler/pull/5456/commits/1f466c04a6d3ea54abfafc4d5fde0a17717ade55

Sorry for the silly mistake.